jscl.lisp

```diff
@@ -145,9 +145,10 @@
 
 (defun read-whole-file (filename)
   (with-open-file (in filename)
-    (let ((seq (make-array (file-length in) :element-type 'character)))
-      (read-sequence seq in)
-      (substitute-if #\Space #'(lambda (char) (eql char #\Return)) seq))))
+    (let* ((seq (make-array (file-length in) :element-type 'character))
+           (char-count (read-sequence seq in))
+           (seq1 (substitute-if #\Space #'(lambda (char) (eql char #\Return)) seq)))
```

Comment:
I assume it is to normalize DOS end of line so the reader can deal with it better? If it is that, it sounds good, but please add a comment 👍 

We could also deal with that in the reader.. not sure if it would be better though.

Reply:
I would suggest closing this PR. 

The solution to the problem of `RETURN` and `NULL` should be common and in one place. That is, in the function of `read` (possible `whatespacesp` predicate).

This will remove the patch with the removal of the `RETURN` in the `load` and solve the problem indicated by the @plonk.